plugins/module_utils/common.py

```diff
@@ -96,6 +98,13 @@
     k8s_import_exception = e
     K8S_IMP_ERR = traceback.format_exc()
 
+try:
+    import jsonpath_rw
```

Comment:
I guess there's probably too much to the jsonpath syntax to not require an additional library, are there other Ansible modules that use this syntax and if so is this the library they use as well?

Reply:
That's was also my concern, I don't really like adding a new library I will try to find if there is such a library already implemented in another plugin

Reply:
I was going to suggest jmespath, too. There is a jmespath filter that used to be in core, but I think recently got moved to community.general. I believe there was some concern about the maintenance of this library, but I'm not sure. jmespath is also a dependency of boto3, so it's at least already a transitive dependency for the aws collection.